data_steward/constants/bq_utils.py

```diff
@@ -0,0 +1,32 @@
+# bq_utils default values
```

Comment:
This is a really nice question, and I've put off answering it until it was the last known thing to address in this PR.  It is a personal preference to have this top level constants package whose directory structure mirrors the directory structure of the project components it is being created for.  There are pros and cons:

Pros:

1.  I like to use the pep 8 import rules structure of Python imports, blank line, third party imports, blank line, and project imports.  I add the comments denoting this too.  Within each block I like to alphabetize the imported components based.  This will visually group all the constants modules imported by another module together.
2.  Having separate constants modules named identical to the module they are used in aids traceability.
3.  A package may contain multiple modules.  If you have only one constants file per package, that can become hard to easily read and maintain, especially if you store a lot of long statements or large structures in them.  This package originally used one constants file.  It grew to over 300 lines.  That is hard to read and maintain. 
4.  Splitting them according to module, means each module only loads what it needs.  So the namespace isn't polluted with a lot of data it isn't using.
5.  If the constants package and structure becomes adopted, it will support inheritance.  We can still define something in one place instead of multiple.

Cons:
1.  Your constants file isn't in the same package you are currently working in.  This might cause a mental shift.
2.  Until the inheritance is worked out, you may still define the same thing multiple times (such as concept ids defined in multiple constants files).
3.  Constants are split into correspondingly named modules.  If it doesn't exist, you have to create it.
4.  If you need constants from more than one module, it is two or more imports.  You will have to use the `import package.module.name as name` convention to distinguish between them.
5.  The mental load of having multiple constants files.

I prefer the package arrangement because it keeps the namespace much less polluted with stuff that a particular module does not need, it is easy to determine what module is using the constant based on the module name, if you are using multiple constants files this gives you an extra idea of how the pieces are working together, and this will make it easier to identify opportunities to use inheritance in the constants files.

I'm available for a discussion on this whenever we want to have one.  I think it would benefit us all to move toward that architectural style.